.github/workflows/javascript-run-nested-promises-parallelly.yml

```diff
@@ -0,0 +1,28 @@
+name: Run nested promises parallelly
```

Comment:
```suggestion
name: Run nested promises in parallel
```

Reply:
Hmmm, thanks! Does it mean there's no word `parallelly`?

Reply:
6cac08a

Reply:
I think so but not sure